daemon/mgr/spec_hook.go

```diff
@@ -0,0 +1,35 @@
+package mgr
+
+import (
+	"context"
+	"strings"
+
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func setupHook(ctx context.Context, c *ContainerMeta, spec *SpecWrapper) error {
+	//if set rich mode, set initscript
+	if c.Config.Rich && c.Config.InitScript != "" {
+		args := strings.Fields(c.Config.InitScript)
+		if args == nil || len(args) == 0 {
```

Comment:
I think `len(args) == 0` is enough, since it already includes the case `args == nil`